core/src/main/java/com/datastrato/gravitino/Configs.java

```diff
@@ -5,17 +5,28 @@
 package com.datastrato.gravitino;
 
 import com.datastrato.gravitino.config.ConfigBuilder;
+import com.datastrato.gravitino.config.ConfigConstants;
 import com.datastrato.gravitino.config.ConfigEntry;
 import java.io.File;
+import org.apache.commons.lang3.StringUtils;
 
 public interface Configs {
 
   String DEFAULT_ENTITY_STORE = "kv";
+  String RELATIONAL_ENTITY_STORE = "relational";
   String ENTITY_STORE_KEY = "gravitino.entity.store";
 
   String DEFAULT_ENTITY_KV_STORE = "RocksDBKvBackend";
   String ENTITY_KV_STORE_KEY = "gravitino.entity.store.kv";
 
+  String DEFAULT_ENTITY_RELATIONAL_STORE = "MySQLBackend";
+  String ENTITY_RELATIONAL_STORE_KEY = "gravitino.entity.store.relational";
+
+  String MYSQL_ENTITY_STORE_URL_KEY = "gravitino.entity.store.mysql.url";
```

Comment:
Should we provide ConfigEntry for these config options?

Reply:
I think them can be provided